### PR TITLE
Added default style for aggregating by admin level

### LINKED ISSUE #10053: Aggregating by admin level should apply an style by value (agg_value_density) by default
Right now it aggregates the data, but it doesnt apply the correct style.

#SMOKE_MG26

### PR DESCRIPTION
**Pull request by xavijam**
Fixes #10053.
CR: @nobuti

### CODE REVIEW

File: lib/assets/javascripts/cartodb3/editor/style/style-defaults/regions-aggregation-style-defaults.js [hunk 1]
```diff
@@ -14,6 +14,19 @@ module.exports = _.defaults({
...
       }
     };
+  },
+
+  _getFillAttrs: function (geometryType) {
+    return {
+      fill: {
+        'color': {
+          attribute: 'agg_value_density',
+          bins: '5',
```
Comment (nobuti) [inline lib/assets/javascripts/cartodb3/editor/style/style-defaults/regions-aggregation-style-defaults.js]: Is there any explanation for these values? Just for learning purposes.

Comment (xavijam) [inline lib/assets/javascripts/cartodb3/editor/style/style-defaults/regions-aggregation-style-defaults.js]: which ones? bins or agg_value_density?

Comment (nobuti) [inline lib/assets/javascripts/cartodb3/editor/style/style-defaults/regions-aggregation-style-defaults.js]: bins and ramp of 5

Comment (xavijam) [inline lib/assets/javascripts/cartodb3/editor/style/style-defaults/regions-aggregation-style-defaults.js]: Those options were suggested by @makella, could you explain it to us @makella 😗 ?

**Pull request merged**